courses/machine_learning/deepdive2/structured/solutions/2_prepare_babyweight.ipynb

```diff
@@ -0,0 +1,834 @@
+{
```

Comment:
<p>Whoops, forgot a comma here too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>These commas after "In this lab" are optional. It will be like this every lab FYI. I'll change it anyway and I guess all of the other labs while I'm at it lol.</p>

Reply:
Sure, it's a small impact, but not quite seeing how good grammar is optional? Unless my grammar is bad and that's not how the English language works? (That is a possibility, that's why I went into math :) )